src/components/ha-selector/ha-selector-select.ts

```diff
@@ -112,7 +113,9 @@ export class HaSelectSelector extends LitElement {
           .disabled=${this.disabled}
           .required=${this.required && !value.length}
           .value=${this._filter}
-          .items=${options.filter((item) => !this.value?.includes(item.value))}
+          .items=${options.filter(
+            (item) => !item.disabled && !this.value?.includes(item.value)
+          )}
```

Comment:
Do we want items filtered and not with disabled state ?

Reply:
We would rather have them disabled, but the combo box doesn't support it...